src/runtime-rs/crates/shim/src/bin/main.rs

```diff
@@ -0,0 +1,192 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use std::{
+    ffi::{OsStr, OsString},
+    path::PathBuf,
+};
+
+use anyhow::{anyhow, Context, Result};
+use nix::{
+    mount::{mount, MsFlags},
+    sched::{self, CloneFlags},
+};
+use shim::{config, Args, Error, ShimExecutor};
+
+const DEFAULT_RUNTIME_WORKER_THREADS: usize = 2;
+const ENV_RUNTIME_WORKER_THREADS: &str = "RUNTIME_WORKER_THREADS";
```

Comment:
`RUNTIME_*` is too generic imho so how about:

```suggestion
const ENV_RUNTIME_WORKER_THREADS: &str = "KATA_RUNTIME_WORKER_THREADS";
```

Reply:
Thanks for your review, since the PR has been merged, I will fix this in a new PR